### DISCUSSION TITLE
kolejny semestr

### DISCUSSION THREAD

**Discussion by magladko** [kpostekk/caats#39] [Q&A] [answered]
pokazują się wam przedmioty z kolejnych semestrów?

**Comment by kpostekk** [comment, marked as answer]
Dane są pobrane do 2023-08-01, jeżeli jest takowa potrzeba to mogę pobrać dane na niecowiększy zakres. Również kwestia wyświetlania przedmiotów jest powiązana  z grupami jakie ma się przypisane

**Reply by magladko** [reply]
chodzi o to, że chciałem plan sobie jakoś ułożyć z obierakami na 7sem., ale poradziłem sobie już w excelu xd.
Natomiast może spoko by było odblokowywać kolejny semestr bliżej końca obecnego, w szczególności w momencie wyboru obieraków, bo to chyba już drugi raz walczyłem xd

**Reply by magladko** [reply]
albo przynajmniej 1. miesiąc kolejnego semestru, żeby tam nie przeciążać niczego

**Reply by kpostekk** [reply]
jak będą w planie zajęć pjakowym zajęcia to caatsach też się pojawią, pewnie jeszcze dziś zwiększę mu zakres pobierania, typu next 4 months, once in a month

**Reply by magladko** [reply]
w sensie na pjatku już są, a u was nie ma

**Reply by kpostekk** [reply]
poważnie? byłem święcie przekonany, że będą dopiero układać plan chwilę przed rozpoczęciem zajęć, zaraz dodam zlecenie scrapu na 180 dni w przód

**Reply by kpostekk** [reply]
w ciągu 12h cały plan będzie raczej zaktualizowany, tak na kolejne 365 dni

**Reply by kpostekk** [reply]
![image](https://github.com/kpostekk/caats/assets/30326322/85150f39-7405-4770-9fba-09c5280fc60d)

Wygląda na to, że już dane na kolejny semestr powinny być

**Reply by magladko** [reply]
są, dzięki. btw coś się psuje w kalendarzu w widoku tygodnia przy przechodzeniu między tygodniami. czasem kliknięcie wstecz przenosi do przodu i vice versa.
